api/db-cdr/generate-cdr/make-bq-data.sh

```diff
@@ -167,11 +167,9 @@ echo "Inserting concept table data ... "
 bq --quiet --project=$BQ_PROJECT query --nouse_legacy_sql \
 "INSERT INTO \`$WORKBENCH_PROJECT.$WORKBENCH_DATASET.concept\`
 (concept_id, concept_name, domain_id, vocabulary_id, concept_class_id, standard_concept,
-concept_code, valid_start_date, valid_end_date, invalid_reason, count_value, prevalence)
```

Comment:
See above, we should keep this.

Reply:
The valid_start_date was being fetched to the count_value reason being inconsistent columns changed this query.